### PR TITLE
Production notes for publishing-bot instance

### PR DESCRIPTION
**Pull request by dims**
Document the usual process to access the publishing-bot instance

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: production.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+### How do i see what the publishing bot is doing?
+
+you can stream the logs of the pod to see what the publishing-bot is doing
+```shell
+kubectl -n publishing-bot logs pod/publisher-cdvwj -f
+```
+
+### What is the persistent volume for?
+
+To do its work the publishing-both has to download all the repositories and performs git surgery on them. So publishing-bot
```
Comment (ameukam) [inline production.md:48]:
```suggestion
To do its work the publishing-bot has to download all the repositories and performs git surgery on them. So publishing-bot
```

**Comment by dims** [comment]
@nikhita on it!

**Comment by dims** [comment]
@nikhita done!

File: production.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+NAME                  READY   STATUS    RESTARTS   AGE
+pod/publisher-cdvwj   1/1     Running   0          9h
+````
+
+### How do i see what the publishing bot is doing?
+
+you can stream the logs of the pod to see what the publishing-bot is doing
+```shell
+kubectl -n publishing-bot logs pod/publisher-cdvwj -f
+```
```
Comment (sttts) [inline production.md:78]: worth to link to the bot issue in k/k?

File: production.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### What is the persistent volume for?
+
+To do its work the publishing-bot has to download all the repositories and performs git surgery on them. So publishing-bot
+keeps the downloaded copy around and re-uses them. For example, if the pod gets killed the new pod can still work off
+of the downloaded git repositories on the persistent volume. Occasionally if we suspect the downloaded git repos are
+corrupted for some reason (say github flakiness), we may have to cleanup the pv/pvc.
+
```
Comment (sttts) [inline production.md:86]: would be even clearer: "The volume is cache only. Wiping it is not harmful (other than for the time it takes to recreate all the data)"

File: production.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+# Kubernetes publishing-bot production instance notes
+
+### What is this and what does it do?
+
+The publishing-bot for the Kubernetes project is running in the `publishing-bot` namespace  on a CNCF sponsored GKE cluster `aaa` in the `kubernetes-public` project.
+The bot is responsible for updating `go.mod` for target repos.
```
[the remark this answers is not in the document]
Comment (dims) [reply production.md:6]: LOL yes, let me just drop this. folks can always read the main README

[the remark this answers is not in the document]
Comment (dims) [reply production.md:6]: note that this was copy-paste from the existing markdown file :)

File: production.md (hunk 2 above)
Comment (dims) [reply production.md:78]: Added a new question+answer for the bot issue in k/k

File: production.md (hunk 3 above)
Comment (dims) [reply production.md:86]: added the sentence at the end.

**Comment by dims** [comment]
@sttts @nikhita ready to merge!

thanks,
dims

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

Let's merge and iterate if needed. This is a great start! :tada:

**Pull request merged**